MozillaFoundation/foundation.mozilla.org
# Convert snippets menu into groups

# Description



~~:warning: Depends on #11726! Please rebase before merging this one! :warning:~~ Rebasing done

This PR converts the snippets menu into categories/groups in Wagtail's menu.

It also converts the deprecated `ModelAdmin` usage into the new `SnippetViewSet`.

To test:
- Log in to the CMS
- See that the menu bar on the left is different and grouped into categories
- Verify that all snippets have been accounted for

Link to sample test page:
Related PRs/issues: #11753 

# Checklist



**Tests**

**Changes in Models:**

**Documentation:**

**Merge Method**
**💡❗Remember to use squash merge when merging non-feature branches into `main`**
